flowapi/flowapi/user_model.py

```diff
@@ -66,8 +73,8 @@ def can_run(self, *, query_json: dict) -> bool:
             If the user cannot run this kind of query at this level of aggregation
 
         """
-
-        return self.has_access(actions=["run"], query_json=query_json)
+        claims = await query_to_scopes(query_json)
```

Comment:
A bit confusing that we use both "claims" and "scopes" here.

Reply:
My thinking specifically for here:
`scopes` are what a user, via a role, *has* access to.
`claims` are what a user, via a query, is *trying* to access
There's ambiguity in that they both come in the same form (in this case, the output from query_to_scopes)

Having read the stackexchange link a few comments down, I realise that this is he exact opposite way around to what it should be - I'll switch them, and maybe refactor `query_to_scopes` to mean something else

Reply:
In a JWT the "claims" are all the fields in the token body - in our case this includes the permission scopes, but also `"sub"`, `"exp"` etc.

I'd suggest using the term "scopes" everywhere here, whether you're referring to the scopes derived from a query spec or the scopes extracted from the JWT claims.